doc/downloading.md

```diff
@@ -0,0 +1,27 @@
+# Downloading SPIRE
+
+This page describes the options available for downloading SPIRE.
+
+## Docker Compose
+
+* [SPIRE 101](SPIRE101.md) is an introduction to SPIRE that runs on Docker Compose
+
+* Additional Docker Compose demos are available in the [spire-tutorials](https://github.com/spiffe/spire-tutorials) repo
+
+## Kubernetes
+
+* No official Helm chart, Kustomize file, or custom resource operator is available for SPIRE but [Quickstart for Kubernetes](https://spiffe.io/docs/latest/spire/installing/getting-started-k8s/) includes a basic set of Kubernetes YAML files for testing SPIRE Server and Agent
+
+* Additional Kubernetes demos are available in the [spire-tutorials](https://github.com/spiffe/spire-tutorials) repo
+
+## Linux Executables and Source
+
+* The SPIRE GitHub [releases](https://github.com/spiffe/spire/releases) page has download links and changelogs for each SPIRE release
+
+* The spiffe.io [Get SPIRE](https://spiffe.io/downloads/) page has additional download options and instructions for building SPIRE yourself
+
+* [Quickstart for Linux and MacOS X](https://spiffe.io/docs/latest/spire/installing/getting-started-linux-macos-x/) describes how to download and test a simple one-node installation of the SPIRE Server and Agent
+
+## MacOS Source
```

Comment:
I'm not sure if this is the right title for this section, since it seems that what we want to describe is that we don't have pre-built SPIRE executables available for MacOS. So maybe `MacOS executables` would be a better choice?